tests/_resources/base_import/content/__metadata__.json

```diff
@@ -22,6 +22,20 @@
       "default_page_uuid": "45b0b46f17104a7b8fa7bb94d3dd5bd9"
     }
   },
+  "local_permissions": {
+    "35661c9bb5be42c68f665aa1ed291418": {
+      "plone.app.contenttypes: Add Image": {
+        "roles": ["Manager"],
+        "acquire": false
+      }
+    },
+    "e7359727ace64e609b79c4091c38822a": {
+      "plone.app.contenttypes: Add Image": {
+        "roles": ["Member"],
+        "acquire": false
+      }
+    }
+  },
```

Comment:
Why does this go in `__metadata__.json` instead of in the folder for this content item? I would think the latter would be more convenient / easier to see which item it is for.

Reply:
I'm following the same pattern used for local roles.

Reply:
Well, it also seems like the wrong place for local roles to me.

Reply:
We have this nice structure where we can add multiple files about each content item, but we aren't using it for some properties that are clearly related to a single content item? It just doesn't make sense.

Reply:
@ericof This doesn't need to block merging this PR; we can move it later. In general I think it'd make sense to keep __metadata__.json for things that are related to the export as a whole and not related to a specific content item.